pkg/runner/run_context.go

```diff
@@ -150,6 +183,21 @@ func (rc *RunContext) startJobContainer() common.Executor {
 		)(ctx)
 	}
 }
+
+const defaultNetwork = "act_github_actions_network"
+
+func (rc *RunContext) createNetwork() common.Executor {
+	return func(ctx context.Context) error {
+		return container.NewDockerNetworkCreateExecutor(defaultNetwork)(ctx)
+	}
+}
+
+func (rc *RunContext) removeNetwork() common.Executor {
+	return func(ctx context.Context) error {
+		return container.NewDockerNetworkRemoveExecutor(defaultNetwork)(ctx)
+	}
+}
+
```

Comment:
If another workflow references same services, there will be a clash.
Perhaps network should be named `act-<workflow-name>-network`